### ISSUE TITLE
Failing to create table due to duplicate paddingOctets IE

### ISSUE THREAD

**Issue by cameronkerrnz** [cameronkerrnz/libipfix#1]
Sample from logs

```
...
  field25: ie=0.210, len=1 (paddingOctets)
  field26: ie=0.210, len=2 (paddingOctets)
...
[ipfix_db] CREATE TABLE ipfix_14 ( ... , ie0_d2 VARBINARY(4096) , ie0_d2 VARBINARY(4096) , ...  )
[ipfix_db] mysql_query() failed: Duplicate column name 'ie0_d2'
[export_trecord_db] cannot build table name for template ...
```

**Comment by cameronkerrnz** [comment]
Impact of change. We can make a simplification and say that the only thing that would reasonably be present duplicated are paddingOctets (0.210).
- Change `ipfix_db_create_table` to introduce a condition to check for IE 210 in ENO 0, and if so, skip the generation of that part of the SQL statement. That should take care of the schema generation (ie. CREATE TABLE)
- But could also adjust `ipfix_db_get_tablename` as the FTE is also present as the table name (sorted FTEs), but that is unnecesary and the logic in that function is not very simple. They would appear in existant column names anyway.
- We also need to make a similar change to the population of the record... (the INSERT INTO)
  I think this is in `ipfix_export_drecord_db`

Such tables would never have been created, so we don't need to worry about removing any such old tables.

**Comment by cameronkerrnz** [comment]
Post change, I can verify that such tables are created:

```
mysql> select table_name, table_rows from information_schema.tables where table_schema = 'netscaler_ipfix';
+------------------------------------------------------+------------+
| table_name                                           | table_rows |
+------------------------------------------------------+------------+
| ipfix_1                                              |          0 |
| ipfix_13                                             |          0 |
| ipfix_14                                             |          0 |
| ipfix_15                                             |          0 |
| ipfix_16                                             |          0 |
| ipfix_17                                             |          0 |
| ipfix_18                                             |          0 |
| ipfix_2                                              |          2 |
| ipfix_22                                             |          0 |
| ipfix_23                                             |          0 |
| ipfix_25                                             |          0 |
| ipfix_26                                             |          0 |
| ipfix_3                                              |          2 |
| ipfix_4                                              |          0 |
| ipfix_4_3c_8a_90_94_d2_v173f_85_c8_f0_f1_f5_f7_f8_f9 |          0 |
| ipfix_4_3c_8a_90_94_d2_v173f_85_c8_f2_f7_f8_f9       |          0 |
| ipfix_5                                              |          0 |
| ipfix_6                                              |          0 |
| ipfix_7                                              |          2 |
| ipfix_8                                              |          0 |
| ipfix_8a_90_v173f_81_139_13a_13b_13c                 |          0 |
| ipfix_8a_90_v173f_81_13d_13e                         |          0 |
| ipfix_8a_90_v173f_81_84                              |          1 |
| ipfix_8a_90_v173f_fb_fc_fd                           |        312 |
| ipfix_8a_v173f_10c_10d                               |          0 |
| ipfix_90_v173f_81_a3_a8                              |          0 |
| ipfix_90_v173f_81_a4_a5                              |          0 |
| ipfix_95_v173f_96_97_98_a7                           |       2568 |
| ipfix_d2_v173f_86_87_88                              |          0 |
| ipfix_exporters                                      |          3 |
| ipfix_mapping                                        |        949 |
| ipfix_messages                                       |       2378 |
| ipfix_templates                                      |         30 |
| ipfix_v17                                            |        312 |
+------------------------------------------------------+------------+
34 rows in set (0.00 sec)
```

**Comment by cameronkerrnz** [comment]
Fixed in bf1f0a90d31e58df86be104d928bd4f8e07bffeb
